### ISSUE TITLE
`react-router-dom` attempting to load packages that don't exist in the npm package

### ISSUE THREAD

**Issue by alex-wilmer** [ReactTraining/react-router#4396]
## Version
4.0.0-beta.2

## Test Case
http://www.webpackbin.com/4JlovKOwG

## Steps to reproduce
install `react-router-dom@4.0.0-beta.2`
import anything from its `index.js`

## Expected Behavior
not to throw an error

## Actual Behavior
complains about not being able to find modules

_____

looking at index.js. It's importing a bunch of stuff that doesn't get included in the root folder.

```
var _BrowserRouter2 = require('./BrowserRouter');

var _BrowserRouter3 = _interopRequireDefault(_BrowserRouter2);

var _HashRouter2 = require('./HashRouter');

var _HashRouter3 = _interopRequireDefault(_HashRouter2);

var _Link2 = require('./Link');

var _Link3 = _interopRequireDefault(_Link2);

var _MemoryRouter2 = require('./MemoryRouter');

var _MemoryRouter3 = _interopRequireDefault(_MemoryRouter2);

var _NavLink2 = require('./NavLink');

var _NavLink3 = _interopRequireDefault(_NavLink2);

var _Prompt2 = require('./Prompt');

var _Prompt3 = _interopRequireDefault(_Prompt2);

var _Redirect2 = require('./Redirect');

var _Redirect3 = _interopRequireDefault(_Redirect2);

var _Route2 = require('./Route');

var _Route3 = _interopRequireDefault(_Route2);

var _Router2 = require('./Router');

var _Router3 = _interopRequireDefault(_Router2);

var _StaticRouter2 = require('./StaticRouter');

var _StaticRouter3 = _interopRequireDefault(_StaticRouter2);

var _Switch2 = require('./Switch');

var _Switch3 = _interopRequireDefault(_Switch2);

var _matchPath2 = require('./matchPath');

var _matchPath3 = _interopRequireDefault(_matchPath2);

var _withRouter2 = require('./withRouter');

var _withRouter3 = _interopRequireDefault(_withRouter2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

`node_modules/react-router-dom` folder

```
react-router-dom (router4beta)*$ ls
BrowserRouter.js	README.md
HashRouter.js		index.js
Link.js			package.json
NavLink.js		umd
```

**Comment by alex-wilmer** [comment]
ahh probably a duplicate, sorry https://github.com/ReactTraining/react-router/issues/4394

**Comment by alex-wilmer** [comment]
well.. maybe it's not, so I'll let you guys close it if it indeed is a dupe

**Comment by pshrmn** [comment]
Dupe of a different one #4391 :)

**Comment by YasserKaddour** [comment]
If you are eager to use the beta; you can move the missing file from `node_modules/react-router` to `node_modules/react-router-dom`, and renaming `ServerRouter.js` to `StaticRouter.js`

**Comment by mjackson** [comment]
Published in 4.0.0-beta.3

    npm install react-router-dom@next
